targets/ath79-generic

```diff
@@ -131,6 +131,8 @@ device('tp-link-eap225-outdoor-v1', 'tplink_eap225-outdoor-v1', {
        packages = ATH10K_PACKAGES_QCA9888,
 })
 
+device('tp-link-tl-wr1043n-nd-v3', 'tplink_tl-wr1043nd-v3')
```

Comment:
Is `tp-link-tl-wr1043n-nd-v3` what is reported as the image name by libplatforminfo? I'd expect this to use the model name from the device tree, in which case it would be "tp-link-tl-wr1043nd-v3", but maybe I'm looking at the wrong string.

Reply:
platforminfo confirmed it, you were right.
resolved in 952f818a0992679e0f602773386aa0323c51e08c